cmd/world/main.go

```diff
@@ -35,5 +60,36 @@ func main() {
 		}()
 	}
 
+	// Posthog Initialization
+	posthog.Init(PosthogApiKey)
+	defer posthog.Close()
+
+	// Obtain the machine ID
+	machineID, err := machineid.ProtectedID("world-cli")
+	if err != nil {
+		logger.Error(err)
+	}
+
+	// Create capture event for posthog
+	event := ph.Capture{
+		DistinctId: machineID,
+		Timestamp:  time.Now(),
+		Properties: map[string]interface{}{
+			"version": AppVersion,
+			"command": os.Args,
+		},
+	}
+
+	// Capture event post installation
+	if len(os.Args) > 1 && os.Args[1] == "post-installation" {
+		event.Event = postInstallationEvent
+		posthog.CaptureEvent(event)
+		return
+	}
+
+	// Capture event running
+	event.Event = runningEvent
+	posthog.CaptureEvent(event)
```

Comment:
We only need this data logged at most once a day. Otherwise, we should probably avoid it since it adds additional execution time to the commands.

Maybe we should use a file to store what date was the event last logged and only log if the last logged day is not today.

Reply:
updated